Update versioning v2 Landmark.mdx (#5486)
grommet/hpe-design-system: aries-site/src/pages/design-tokens/versioning.mdx

## Production-ready versions

### ^2.0.0

> This release (known as Landmark) provides production-ready support for HPE's refreshed visual identity (October 2025 - present). It introduces the new Landmark color palette and the Graphik font face to align with the updated brand expression. This version aligns with [grommet-theme-hpe version 8](https://github.com/grommet/grommet-theme-hpe/releases/tag/v8.0.0)

Reviewer: ```suggestion
This release, referred to as "v2-Landmark," provides production-ready support for HPE's refreshed visual identity (October 2025 - present). It introduces the new Landmark color palette and the HPE Graphik font face to align with the updated brand expression.

For Grommet applications, v2-Landmark is available by [grommet-theme-hpe version 8](https://github.com/grommet/grommet-theme-hpe/releases/tag/v8.0.0)
```

In the [HPE Design System Figma Component Library](https://www.figma.com/design/HDckqS2MWhINfC8EIQPMV1/HPE-Design-System-Components-V2?m=auto&t=NjHwxPFWYTvhZTP5-6), this is the default theme.

### ^1.0.0

> This release provides production-ready support with styles in alignment with the latest HPE theme (February 2025 - present). This version aligns with grommet-theme-hpe version 6 to 7.

Author: What do you think if we make the description such that it doesn't have to be edited here on out. We can place this notice 

> "This HPE theme was introduced in February 2025. Teams should coordinate with their product management and HPE brand for upgrade timelines to v2-Landmark." 

in an Inline informational notification. What do you think?

Update would look something like this:
(note the date formatting in the brackets would be used across each description too)

^1.0.0

<notification>
This HPE theme was introduced in February 2025. Teams should coordinate with their product management and HPE brand for upgrade timelines to v2-Landmark." in an Inline informational notification. What do you think? </notification>

This release provides production-ready support with styles in alignment with the HPE theme (February 2025 - October 2025).

For Grommet applications, v1 is made available in grommet-theme-hpe versions [6.x](https://github.com/grommet/grommet-theme-hpe/releases/tag/v6.0.0) - [7.x](https://github.com/grommet/grommet-theme-hpe/releases/tag/v7.0.0).

## Production-ready versions

### ^2.0.0

This release, referred to as "v2-Landmark," provides production-ready support for HPE's refreshed visual identity (October 2025 - present). It introduces the new Landmark color palette and the HPE Graphik font face to align with the updated brand expression.

For Grommet applications, v2-Landmark is available by [grommet-theme-hpe v8.x](https://github.com/grommet/grommet-theme-hpe/releases/tag/v8.0.0)

In the [HPE Design System Figma Component Library](https://www.figma.com/design/HDckqS2MWhINfC8EIQPMV1/HPE-Design-System-Components-V2?m=auto&t=NjHwxPFWYTvhZTP5-6), this is the default theme.

### ^1.0.0

<Notification
  status="info"
  message="This HPE theme was introduced in February 2025. Teams should coordinate with their product management and HPE brand for upgrade timelines to v2-Landmark."
  actions={[
    { label: 'All design tokens', href: '/design-tokens/all-design-tokens' },
  ]}
  width={{ max: 'xlarge' }}
/>

> This HPE theme was introduced in February 2025.

Reviewer: Can we try to add whitespace between the notification and the following sentence? Not sure why its not including it now, but you may be able to add a `<br />` to address.

<img width="840" height="363" alt="Screenshot 2025-11-05 at 8 41 35 AM" src="https://github.com/user-attachments/assets/c910e2c1-2842-4cc2-ad74-2077cdd20d6b" />

  message="To ease adoption of hpe-design-tokens, it is recommended to use v0.9.0 for initial mapping of tokens to local components. This will achieve style parity with an application's current look and allow quality assurance against familiar styling. When ready, teams can simply upgrade to v1 to view the latest theme styles."
/>

## Migration

> ### Migrating from v1.0.0 to v2.0.0

Reviewer: No need for the additional version precision. Major version is adequate.

```suggestion
### Migrating from v1 to v2
```

> v2.0.0 (Landmark) introduces significant updates to align with HPE's refreshed visual identity. This is a major version change reflecting a new brand expression and includes breaking changes.

Reviewer: Redundant with the ### ^2.0.0 section above.

```suggestion
v2-Landmark introduces significant updates to align with HPE's refreshed visual identity. This is a major version change reflecting a new brand expression and includes breaking changes.
```

> A full list of changes and detailed instructions can be found in the Migrating from [hpe-design-tokens v1 to v2-Landmark guide](https://github.com/grommet/hpe-design-system/wiki/Migrating-from-hpe%E2%80%90design%E2%80%90tokens-v1-to-v2%E2%80%90Landmark) and the [grommet-theme-hpe v7 to v8 Migration Guide](https://github.com/grommet/grommet-theme-hpe/wiki/Migrating-from-HPE-Theme-7.x-to-8.x).

Reviewer: ```suggestion
A full list of changes and detailed instructions can be found in the [Migrating from hpe-design-tokens v1 to v2-Landmark guide](https://github.com/grommet/hpe-design-system/wiki/Migrating-from-hpe%E2%80%90design%E2%80%90tokens-v1-to-v2%E2%80%90Landmark) and the [grommet-theme-hpe v7 to v8 Migration Guide](https://github.com/grommet/grommet-theme-hpe/wiki/Migrating-from-HPE-Theme-7.x-to-8.x).
```
